horace_core/sqw/@sqw/display.m

```diff
@@ -1,4 +1,4 @@
-function display(w)
+function w=display(w)
```

Comment:
what changes that you need to return w?

Reply:
1) for caching some values -- may be especially useful for filebacked objects.
2) I was thinking about identifying output type using build-in disp function. Have not succeeded but this is the result of this thinking.